docs/README.md

```diff
@@ -0,0 +1,114 @@
+# Dispatch-Proxy Docs
+
+Dispatch-proxy is a SOCKS5/HTTP proxy that balances traffic between multiple internet connections.
+
+## Pre-requisites
+* Mac OS X 10.6+
+* Windows 7+
+* Node.JS >= 0.10.0
+
+## Installation instructions
+
+* Windows: [imgur album](http://imgur.com/a/0snis)
+* Mac OS X: [imgur album](http://imgur.com/a/TSD5F)
+
+## Help and Support
+Please check out the documentation and FAQ first. We do not offer support but if you find a bug, you are more than welcome to open an issue.
+
+### FAQ: ###
+**Q: Can I run on a router?**
+
+**A:** It would be difficult to run this proxy on a wireless router, including a router configured as repeater because most of the routers on the market can only work at AP mode or Repeater mode. Thus a router generally saying cannot route packets through Ethernet and wireless connection to another AP at the same time.
+
+It has been a long time since you asked. But if you are still interested, I strongly encourage you to test running this project on a router which you have ssh access to. Please do let us know if any success!
```

Comment:
Maybe remove the first sentence/reword this so it makes more sense in a FAQ:

> We strongly encourage you to test running this project on a router which you have ssh access to. Please do let us know if any success!

Reply:
Oh sorry, copy and pasted from an issue.

Reply:
Did you mean to remove the whole first paragraph?

Reply:
No, just reword the second paragraph.